posthog/hogql_queries/query_runner.py

```diff
@@ -860,15 +860,19 @@ def run(
             if self.is_query_service:
                 tag_queries(chargeable=1)
 
-            fresh_response_dict = {
-                **self.calculate().model_dump(),
-                "is_cached": False,
-                "last_refresh": last_refresh,
-                "next_allowed_client_refresh": last_refresh + self._refresh_frequency(),
-                "cache_key": cache_key,
-                "timezone": self.team.timezone,
-                "cache_target_age": target_age,
-            }
+            with get_app_org_rate_limiter().run(
+                org_id=self.team.organization_id,
+                task_id=self.query_id,
```

Comment:
add `team_id=self.team.id`, it will be used for metrics

Reply:
metric: posthog_clickhouse_query_concurrency_limit_exceeded

the second chart here:
https://grafana.prod-us.posthog.dev/d/cef026mzzr7y8b/api-query-concurrent-rate-limiting?var-team_id=$__all